### PR TITLE
Added support for new golang logs with unix timestamp

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/interpreter.js [hunk 1]
```diff
@@ -173,15 +173,19 @@ function created(utcSeconds) {
...
-  var date = new Date(ts)
-  var dateString =
-    MONTHS[date.getUTCMonth()] + ' ' +
-    padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
-    padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
-    padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
-    padLeft(date.getUTCSeconds().toString(), 2, '0')
-
-  return `<span class="created">${padSpacesRight((date.getTime() / 1000).toString(), 14)} </span><span class="date">${dateString} </span>`
+  if (typeof(ts) == "string") {
```
Comment (guykr) [inline src/interpreter.js]:
```suggestion
  if (typeof ts === "string") {
```

File: src/interpreter.js [hunk 2]
```diff
@@ -173,15 +173,19 @@ function created(utcSeconds) {
...
+    var dateString =
+      MONTHS[date.getUTCMonth()] + ' ' +
+      padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
+      padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
+      padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
+      padLeft(date.getUTCSeconds().toString(), 2, '0')
+
+    return `<span class="created">${padSpacesRight((date.getTime() / 1000).toString(), 14)} </span><span class="date">${dateString} </span>`
+  } else {
+    return created(ts)
```
Comment (guykr) [inline src/interpreter.js:183]: not sure what/why we have two almost identical functions, but I guess that's out of the scope of this fix.
Perhaps a little nicer it so have single `created` function that handles both cases (number and string)?

Comment (ghost) [reply src/interpreter.js:183]: extracted the common code to a function

File: src/interpreter.js (hunk 1 above)
Comment (ghost) [reply src/interpreter.js]: done

File: src/interpreter.js [hunk 3]
```diff
@@ -162,26 +162,26 @@ function keyVal(value) {
...
-  var date = new Date(ts)
-  var dateString =
-    MONTHS[date.getUTCMonth()] + ' ' +
-    padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
-    padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
-    padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
-    padLeft(date.getUTCSeconds().toString(), 2, '0')
-
-  return `<span class="created">${padSpacesRight((date.getTime() / 1000).toString(), 14)} </span><span class="date">${dateString} </span>`
+  if (typeof ts  == "string") {
```
Comment (guykr) [inline src/interpreter.js]: just replace `==` with `===` (it is more performant)

**Review by guykr — APPROVED** [review]

File: src/interpreter.js (hunk 3 above)
Comment (ghost) [reply src/interpreter.js]: done

**Pull request merged**